P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -1729,12 +1730,47 @@ be marked with the <gi>supplied</gi> and <gi>surplus</gi> elements
 <gi>del</gi>. These two sets of elements allow the encoder to
 distinguish editorial changes from those visible in the source text.
 </p>
+<p>The <gi>ellipsis</gi> element provides a means to encode the deliberate, marked omission of content in a source document. 
+  Within <gi>ellipsis</gi>, <gi>metamark</gi> describes the denotation of omitted material in the source, and encoders may optionally provide 
+  a <gi>note</gi> describing the omission and <gi>supplied</gi> to indicate the content omitted at this point. We provide an example of such an indicated omission from a manuscript of song lyrics.
+  <figure><graphic url="Images/cod-10991.jpg" mimeType="image/jpeg"/>
+    <head>A manuscript from Cancioneiro da Biblioteca Nacional [1525-1526] in Biblioteca Nacional de Portugal, representing song lyrics and the beginning of a chorus. On the last line visible in this image, a marking of three dots and a line, ·:—, indicates that the rest of the chorus should follow after this point.</head>
+  </figure>
+Our encoding of the passage shows the use of <gi>ellipsis</gi> with <gi>supplied</gi> to hold the omitted content.
+  <egXML xmlns="http://www.tei-c.org/ns/Examples" xml:lang="gl" source="#COEDADD-eg-93">
+    <lg>
+      <l>Amıgas sey eu be<ex>n</ex> dunha molher</l>
+      <l>Que se trabalha de uosco buscar</l>
+      <l>Mal a uossamigo polo matar</l>
+      <l>Mays todaquestamiga ela q<ex>ue</ex>r</l>
+      <metamark function="signalChorus">꜒</metamark>
+      <l>Por q<ex>ue</ex> nu<ex>n</ex>ca co<ex>n</ex> el pode poer</l>
+      <l>Queo podesse por amigauer</l>
+    </lg>
+    <lg>
+      <l>E buscalhi co<ex>n</ex> uosco q<ex>ua</ex>nto mal</l>
+      <l>Ela mays pode aq<ex>ue</ex>sto sei eu</l>
+      <l>E todaq<ex>ue</ex>stela faz polo seu</l>
+      <l>E p<ex>or</ex>este p<unclear>ẏ</unclear>te non por al</l>
+      <ellipsis>
```

Comment:
@syd's suggestion
```xml
<metamark function="signalChorus">꜒</metamark>
<l rend="nobreak" part="I">Por q<ex>ue</ex> nu<ex>n</ex>ca</l>
<ellipsis>
  <metamark rend="nobreak" function="ellipsis">·:—</metamark>
    <supplied>
      <l style="nobreak" part="F">co<ex>n</ex> el pode poer</l>
      <l style="break">Queo podesse por amigauer</l>
    <supplied>
<ellipsis>
```
may be better.

Reply:
From today's conversation, `<metamark>` is not semantically part of a `<l>`, so it doesn't need to be inside it. And it is preferable for `<metamark function="ellipsis">` to be the first child of `<ellipsis>` to remove ambiguity, especially in processing.